### PR TITLE
feat: Change the source route for apps and networks

### PR DESCRIPTION
**Pull request by jinoosss**

### CODE REVIEW

File: packages/adena-extension/src/resources/chains/chains.json [hunk 1]
```diff
@@ -0,0 +1,59 @@
+[
+  {
+    "id": "test3",
+    "main": true,
```
Comment (r3v4s) [inline packages/adena-extension/src/resources/chains/chains.json:4]: key `main` is little bit confused with blockchain terms `mainnet`, `testnet`

**Review by r3v4s — APPROVED** [review]
one small comment

File: packages/adena-extension/src/resources/chains/chains.json (hunk 1 above)
Comment (jinoosss) [reply packages/adena-extension/src/resources/chains/chains.json:4]: How about 'default'?

Comment (r3v4s) [reply packages/adena-extension/src/resources/chains/chains.json:4]: works for me

**Pull request merged**